src/main/webapp/WEB-INF/views/UserPageVer1.2.html

```diff
@@ -0,0 +1,373 @@
+                    <div class="cta-desc">
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-7">
+                    <div>
+                        <form class="form-horizontal" action="" method="" id="addInterest">
+                            <div class="col-md-6">
+                                <select id="addInterestOption" class="form-group select-input" title="What do you want to invest in?">
+                                    <option disabled selected>What do you want to invest in?</option>
+                                    <option value="1">Startups and Investment Projects</option>
+                                    <option value="2">Venture capital</option>
+                                    <option value="3">Mergers and acquisitions</option>
+                                </select>
+                            </div>
+                            <div class="col-md-6 cta-button">
+                                <button type="submit" class="btn btn-lg btn-block btn-info">Add new Interest</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="bs-calltoaction bs-calltoaction-success">
+            <div class="row">
+                <div class="col-md-5 cta-contents">
+                    <h1 class="cta-title">Add new Event</h1>
+                    <div class="cta-desc">
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-7">
+                    <form class="form-horizontal" action="" method="" id="addEvent">
+                        <fieldset>
+                            <div class="form-group col-md-6">
+                                <div class="col-md-12">
+                                    <input id="EventName" name="EventName" type="text" placeholder="Enter event name"
+                                           class="form-control input-md input-text">
+                                </div>
+                            </div>
+                            <div class="col-md-6 cta-button cta-form">
+                                <button type="submit" class="btn btn-lg btn-block btn-success">Add Event</button>
+                            </div>
+                        </fieldset>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+
+</div>
+<div class="footer">
+    <div class="container">
+        <div class="row centered">
+            <a href="#"><i class="fa fa-twitter"></i></a>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)